.storybook/stories/pagination.stories.js

```diff
@@ -24,20 +24,136 @@ export default {
+    changePage(page);
+  };
+  const resultsText = (
+    <div className="usa-dt-pagination__totals">
+      <strong>Custom</strong> results text
+    </div>
+  );
+  return (
+    <div className="story__container">
+        <Pagination
+            changePage={handlePageChange}
+            currentPage={page}
+            pageSize={10}
+            totalItems={100}
+            customResultsText={resultsText} />
+    </div>
+  );
+};
+
+export const limitSelectorPagination = () => {
+  const [page, changePage] = useState(1);
+  const handlePageChange = (page) => {
+    console.log("changePage handler invoked w/ this parameter: ", page);
+    changePage(page);
+  };
+  const [pageSize, changeLimit] = useState(10);
+  const handleChangeLimit = (pageSize) => {
+    console.log("changeLimit handler invoked w/ this parameter: ", pageSize);
+    changeLimit(pageSize);
+  }
+  return (
+    <div className="story__container">
+        <Pagination
+            changePage={handlePageChange}
+            currentPage={page}
+            pageSize={pageSize}
+            totalItems={100}
+            limitSelector
+            resultsText
+            changeLimit={handleChangeLimit} />
+    </div>
+  );
+};
+
+export const goToPagePagination = () => {
+  const [page, changePage] = useState(1);
+  const handlePageChange = (page) => {
+    console.log("changePage handler invoked w/ this parameter: ", page);
+    changePage(page);
+  };
+  return (
+    <div className="story__container">
+        <Pagination
+            changePage={handlePageChange}
+            currentPage={page}
+            pageSize={10}
+            totalItems={100}
+            goToPage />
+    </div>
+  );
+};
+
+export const allOptionsPagination = () => {
+  const [page, changePage] = useState(1);
+  const handlePageChange = (page) => {
+    console.log("changePage handler invoked w/ this parameter: ", page);
+    changePage(page);
+  };
+  const [pageSize, changeLimit] = useState(10);
+  const handleChangeLimit = (pageSize) => {
+    console.log("changeLimit handler invoked w/ this parameter: ", pageSize);
+    changeLimit(pageSize);
+    changePage(1);
+  }
+  return (
+    <div className="story__container">
+        <Pagination
+            changePage={handlePageChange}
+            currentPage={page}
+            pageSize={pageSize}
+            totalItems={100}
+            goToPage
+            limitSelector
+            resultsText
+            changeLimit={handleChangeLimit} />
+    </div>
+  );
+};
+
 defaultPagination.story = { name: 'Default Pagination' };
+resultsTextPagination.story = { name: 'Pagination with default results text' };
+customResultsTextPagination.story = { name: 'Pagination with custom results text' };
+limitSelectorPagination.story = { name: 'Pagination with limit selector' };
+goToPagePagination.story = { name: 'Pagination with "go to page"' };
+allOptionsPagination.story = {name: 'Paginatino with limit selector and "go to page"'}
```

Comment:
Can we make these story names more concise? This will already be under the Pagination name space so we could just do:

w/ ____ or maybe not even the "w/ " just `limit selector and go to page` etc...

Also, typo on line `159`